app/controllers/welcome_controller.rb

```diff
@@ -1,6 +1,4 @@
 class WelcomeController < ApplicationController
```

Comment:
We deleted this?

Reply:
Oh never mind - it is white space removal

Reply:
I was a rubocop violation, so I deleted all the white space.